language/tools/move-package/src/resolution/resolution_graph.rs

```diff
@@ -572,6 +595,24 @@ impl ResolvingGraph {
         }
         Ok(())
     }
+
+    fn increase_movey_download_count(movey_url: String, git_info: &GitInfo, skip_movey: bool) {
+        if skip_movey {
+            return;
+        }
```

Comment:
I feel like it is cleaner to move this to outside of this function. Feels awkward to have a bool flag that is just immediately used to return or not

Reply:
This can be resolved depending on the solution of https://github.com/move-language/move/pull/318#discussion_r968187683